client/test/router_store.spec.js

```diff
@@ -125,7 +125,7 @@ const fixtures = {
 };
 
 
-describe.skip('Router Store', () => {
```

Comment:
I see that you removed test that are failing, but you didnt replace it. By the way, I will fix this in PR https://github.com/cozy/cozy-emails/pull/878